breezedecoration.cpp

```diff
@@ -287,19 +287,37 @@ namespace SierraBreeze
     {
         // Konsole config profile path
         const QString configLocation(QDir::homePath() + "/.local/share/konsole/");
+        const QString shellProfileLocation(configLocation + "/Shell.profile");
```

Comment:
On my system, there is no "Shell.profile". You should get the right name from ~/.config/konsolerc

Reply:
Done